packages/components/src/internal/components/container/FolderAPIWrapper.ts

```diff
@@ -6,7 +6,11 @@ import { SAMPLE_MANAGER_APP_PROPERTIES } from '../../app/constants';
 
 export interface ProjectSettingsOptions {
     allowUserSpecifiedNames?: boolean;
-    name: string;
+    disabledAssayDesigns?: number[];
+    disabledDataClasses?: number[];
+    disabledSampleTypes?: number[];
+    disabledStorages?: number[];
```

Comment:
```suggestion
    disabledStorageLocations?: number[];
```
(or perhaps just `disabledLocations`)

Reply:
updated